src/index.html

```diff
@@ -19,9 +19,388 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com/"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com/"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div class="all">
+      <header class="header">
+        <a
+          href="#"
+          class="header__logo"
+        >
+          <img
+            src="/src/images/logo.png"
+            alt="moyo"
+            class="logo-img"
+          />
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#"
+                class="is-active nav__link"
+              >
+                APPLE
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                SAMSUNG
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                SMARTPHONES
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link nav-hover"
```

Comment:
The `nav-hover` class and `data-qa="nav-hover"` seem to be used for testing purposes. If they are not used in the actual styling or scripting, they should be removed.